src/Compilers/CSharp/Portable/Symbols/Extensions/SourceExtensionImplementationMethodSymbol.cs

```diff
@@ -131,6 +135,11 @@ internal override int TryGetOverloadResolutionPriority()
             return UnderlyingMethod.TryGetOverloadResolutionPriority();
         }
 
+        public override string GetDocumentationCommentXml(CultureInfo? preferredCulture = null, bool expandIncludes = false, CancellationToken cancellationToken = default)
```

Comment:
>GetDocumentationCommentXml [](http://example.com/codeflow?start=31&length=26)

It would be good to have direct tests for this API.

Reply:
There's multiple direct tests (` XmlDoc_01` through ` XmlDoc_03`). Do you have specific API tests in mind?

Reply:
> Do you have specific API tests in mind?

Yes, the effect of ```expandIncludes``` and effect of caching, including on consecutive calls with different ```expandIncludes```.

Reply:
Added an assertion